commitment/server/fetch_repo.ts

```diff
@@ -62,6 +65,12 @@ Meteor.methods({
   },
 });
 
+// can have a case here to see if it is deployment or a docker localhost
+// this means that the API can be connected to without the connection being hard coded
+const DEV_API_CONN_ENDPOINT = "haskell-api:8081";
```

Comment:
Could you please have this read from a `.env` file, once that is addressed this can be merged.